collector-http/src/main/java/zipkin2/collector/otel/http/OpenTelemetryHttpCollector.java

```diff
@@ -52,38 +78,87 @@ public static final class Builder extends CollectorComponent.Builder {
   }
 
   final Collector collector;
+
   final CollectorMetrics metrics;
 
   OpenTelemetryHttpCollector(Builder builder) {
     collector = builder.delegate.build();
     metrics = builder.metrics;
   }
 
-  @Override public OpenTelemetryHttpCollector start() {
+  @Override
+  public OpenTelemetryHttpCollector start() {
     return this;
   }
 
-  @Override public String toString() {
+  @Override
+  public String toString() {
     return "OpenTelemetryHttpCollector{}";
```

Comment:
do we want to name this similar to the sender?

Reply:
Can you elaborate this?